### ISSUE TITLE
[BUG] Update Table Name in rollup_economics_ethereum_l1_fees.sql

### ISSUE THREAD

**Issue by abdelhaqs** [duneanalytics/spellbook#6592]
**Description**: 
The table name in `rollup_economics_ethereum` has changed from `l1_data_fees` to `l1_fees`. which causing broken queries.
Please update the file at `dbt_subprojects/hourly_spellbook/models/_sector/rollup_economics/ethereum/rollup_economics_ethereum_l1_fees.sql`

**Comment by abdelhaqs** [comment]
I understand this [refactor ](https://github.com/duneanalytics/spellbook/pull/6426)focus was on improving performance by restructuring `l1_data_fees` and `l1_verification_fees` into separate SQL files per chain to reduce memory load. It seems the table name change from `l1_data_fees` to `l1_fees` in `rollup_economics_ethereum` might be causing broken queries, so I've created an issue to address this.

**Comment by jeff-dude** [comment]
thank you for raising.

@lgingerich @0xRobin are we fine to go back to original naming standard, or should we find a way to communicate how to fix queries?

**Comment by 0xRobin** [comment]
@abdelhaqs do you have an example of the broken queries?
The `l1_data_fees` model is still present, `l1_fees` is a new model that's added on top that aggregates blob, data and verification fees.

**Comment by abdelhaqs** [comment]
@0xRobin Yes, the l1_data_fees model is still present, but it appears that some column names have changed. For example, in this query: https://dune.com/queries/3897133/, I'm encountering the following error:

Error: Line 5:13: Column 'gas_spent_usd' cannot be resolved at line 5, position 13. Please verify that this column exists using the data explorer.

**Comment by 0xRobin** [comment]
@abdelhaqs indeed, this column is renamed to the following `data_fee_usd`
